packages/docsify-changelog-plugin/src/index.js

```diff
@@ -33,11 +33,213 @@ const plugin = (hook, vm) => {
+    , document.documentElement.style.setProperty(`--changelog-header-h2-text`, `${u.header.h2_text}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h3-text`, `${u.header.h3_text}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h4-text`, `${u.header.h4_text}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h1-background`, `${u.header.h1_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h2-background`, `${u.header.h2_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h3-background`, `${u.header.h3_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h4-background`, `${u.header.h4_background}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h1-size`, `${u.header.h1_size}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h2-size`, `${u.header.h2_size}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h3-size`, `${u.header.h3_size}`)
+    , document.documentElement.style.setProperty(`--changelog-header-h4-size`, `${u.header.h4_size}`)
+    , document.documentElement.style.setProperty(`--changelog-text-all`, `${u.text.all}`)
+    , document.documentElement.style.setProperty(`--changelog-text-strong`, `${u.text.strong}`)
+    , document.documentElement.style.setProperty(`--changelog-text-strong-background`, `${u.text.strong_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-p`, `${u.text.p}`)
+    , document.documentElement.style.setProperty(`--changelog-text-p-background`, `${u.text.p_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-blockquote`, `${u.text.blockquote}`)
+    , document.documentElement.style.setProperty(`--changelog-text-blockquote-background`, `${u.text.blockquote_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-list`, `${u.text.list}`)
+    , document.documentElement.style.setProperty(`--changelog-text-list-background`, `${u.text.list_background}`)
+    , document.documentElement.style.setProperty(`--changelog-text-list-icon`, `${u.text.list_icon}`)
+    , document.documentElement.style.setProperty(`--changelog-link-text`, `${u.link.a_text}`)
+    , document.documentElement.style.setProperty(`--changelog-link-text-hover`, `${u.link.text_hover}`)
+    , document.documentElement.style.setProperty(`--changelog-link-text-transition`, `${u.link.text_transition}`)
+    , document.documentElement.style.setProperty(`--changelog-link-background`, `${u.link.background}`)
+    , document.documentElement.style.setProperty(`--changelog-link-background-hover`, `${u.link.background_hover}`)
+    , document.documentElement.style.setProperty(`--changelog-link-background-transition`, `${u.link.background_transition}`);
+    e.config.changelog = u
```

Comment:
I think `e` is not defined here . 
Please take a look

Reply:
when I change it to `vm.config.changelog`

it shows like this 

![image](https://user-images.githubusercontent.com/26347874/76892322-8437be80-68b0-11ea-8a94-bd35bccb6328.png)

Reply:
this is an issue with me pasting in from my version. I used your minified version as a base to work from.

Reply:
I grabbed the source and am using that as the base now, will make it easier to merge. I had not originally approached this with this scope of involvement, but have set everything up to be more in sync with github.

Reply:
I still need to do testing using the index from this project to test the issue with overlap. I honestly am not sure why it loses its position in the block when it should be the same as before set as absolute. I made adjustments to this in order to make this into a fixed position button instead of stuck at the top of the page.